src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,77 @@
         </div>
     {% endif %}
 
+    <h2>Security Firewall</h2>
+
+    {% if collector.firewall %}
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name }}</span>
+                <span class="label">Name</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.securityEnabled ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Security enabled</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.stateless ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Stateless</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.allowsAnonymous ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Allow Anonymous</span>
+            </div>
+        </div>
+        {% if collector.firewall.securityEnabled %}
+            <table>
+                <thead>
+                    <tr>
+                        <th scope="col" class="key">Key</th>
+                        <th scope="col">Value</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th>provider</th>
+                        <td>{{ collector.firewall.provider }}</td>
+                    </tr>
+                    <tr>
+                        <th>context</th>
+                        <td>{{ collector.firewall.context }}</td>
+                    </tr>
+                    <tr>
+                        <th>entry_point</th>
+                        <td>{{ collector.firewall.entryPoint }}</td>
+                    </tr>
+                    <tr>
+                        <th>user_checker</th>
+                        <td>{{ collector.firewall.userChecker }}</td>
+                    </tr>
+                    <tr>
+                        <th>access_denied_handler</th>
+                        <td>{{ collector.firewall.accessDeniedHandler }}</td>
+                    </tr>
+                    <tr>
+                        <th>access_denied_url</th>
+                        <td>{{ collector.firewall.accessDeniedUrl }}</td>
+                    </tr>
+                    <tr>
+                        <th>listeners</th>
+                        <td>{{ collector.firewall.listeners|yaml_encode }}</td>
+                    </tr>
+                </tbody>
+            </table>
+        {% endif %}
+    {% elseif collector.enabled %}
+        <div class="empty">
+            <p>There is no security firewall.</p>
```

Comment:
@iltar This panel only gives informations about the firewall matching the current Request instance (used for the whole collector).

So, this situation can occur in two cases:
- The FirewallConfig instance is not provided by `FirewallConfig::getContext` (very edge case)
- The current request doesn't match any firewall

I can reproduce the second case simply by setting `pattern: ^/foo` and browse `/` for instance.

Maybe I'm wrong and it should always match with a firewall?